packages/http/src/validator/__tests__/index.spec.ts

```diff
@@ -145,6 +145,8 @@ describe('HttpValidator', () => {
   describe('validateOutput()', () => {
     describe('output is set', () => {
       it('validates the body and headers', () => {
+        (findResponseSpecModule.findOperationResponse as jest.Mock).mockReturnValue({ statusCode: 200 });
```

Comment:
Note to self: where is this being spyed/mocked?

Reply:
On the beginning of the file. This is a pure unit test with the whole environment mocked.